release/models/aft/openconfig-aft-common.yang

```diff
@@ -342,6 +348,18 @@ submodule openconfig-aft-common {
         "The protocol from which the AFT entry was learned.";
     }
 
+    leaf preference {
+      type uint32;
+      description
+        "The administrative distance (preference) of the AFT entry.";
+    }
+
+    leaf metric {
+      type uint32;
+      description
+        "The metric (or cost) of the AFT entry.";
```

Comment:
This and the preference seem a little questionable to me -- what's the use case for having this here vs. having it in the LSDB of the protocol that would populate it?

Reply:
As mentioned in the other comment, primary driver is gRIBI programming/flexibility but that aside, much like `show route` and `show route extensive|detail` across implementations - it is common to display entries (all or preferred) with their respective preference/metric

Reply:
Removed AFT related preference/metric in latest commit and adjusted PR header